lifecycle.go

```diff
@@ -132,10 +133,10 @@ func (d LifeCycle) Run(testplanName string) {
 
 		// save workerCount to consul
```

Comment:
Is the main purpose of this anonymous function monitoring container resource usage? If so, should we move `workerCount ` out of this anonymous function?